cmd/esc/cli/cli_test.go

```diff
@@ -1110,8 +1130,14 @@ func loadTestcase(path string) (*cliTestcaseYAML, *cliTestcase, error) {
 			revisions = cliTestcaseRevisions{Revisions: []cliTestcaseRevision{{YAML: env}}}
 		}
 
+		var tags cliTestcaseEnvironmentTags
+		envTags := map[string]string{}
+		if err := env.Decode(&tags); err == nil {
+			envTags = tags.Tags
+		}
```

Comment:
Sorry do you mean moving the existing tags to `cliTestcaseRevisions` instead of renaming them to `revisionTags` on the test environment?

Reply:
Yeah--it looks like as written an environment can either contain a set of tags or a set of revisions (unless I'm reading it wrong?). Feels like instead we want `cliTestcaseRevisions` to contain a field of type `cliTestcaseEnvironmentTags`.

Reply:
ah yeah I follow you - will update

Reply:
actually thinking more about this I feel like it makes sense to keep these at the top level on the env since revision tags must be unique across all revisions. I did adjust the interface of revision tags though to allow for passing a list instead of a single value since that mirrors what the API supports